ovos_dinkum_listener/voice_loop/microphone.py

```diff
@@ -59,27 +35,42 @@ class AlsaMicrophone(Microphone):
     _thread: Optional[Thread] = None
     _queue: "Queue[Optional[bytes]]" = field(default_factory=Queue)
     _is_running: bool = False
+    _lock = Lock()
```

Comment:
this class is also a plugin now, it + pyaudio microphone should be deleted

mic creation should use opm MicrophoneFactory to use a mic pllugin

Reply:
or for now, just add the plugin as a dependency and import alsa microphone from there